Sources/Document/Json/JSONTree.swift

```diff
@@ -267,7 +267,7 @@ public class JSONTree {
     /**
      * `editByPath` edits this tree with the given node and path.
      */
-    public func editByPath(_ fromPath: [Int], _ toPath: [Int], _ content: (any JSONTreeNode)?) throws -> Bool {
+    public func editByPath(_ fromPath: [Int], _ toPath: [Int], _ contents: [any JSONTreeNode]?) throws -> Bool {
```

Comment:
Could be trivial, but there might be a small cost when using existential types.
I suggest using 'some' whenever we can in place of 'any'

reference: https://github.com/apple/swift-evolution/blob/main/proposals/0335-existential-any.md
```Existential types in Swift have significant limitations and performance implications. ```

Reply:
Since editPyPath() accepts nil for contents, We can't compile code with the "some" keyword. 

<img width="989" alt="스크린샷 2023-08-28 오후 3 22 30" src="https://github.com/yorkie-team/yorkie-ios-sdk/assets/3667927/e7ac5ceb-5c4a-4ddf-ad33-2ea5d15c429c">

Reply:
That's strange. Changing any -> some compiles well on my machine. 
Fom my understanding, accepting nil or not seems irrelevant to the subject? (Not sure though)
Could you check if you've changed your function header to the same as below? 

```swift
public func editByPath(_ fromPath: [Int], _ toPath: [Int], _ contents: [some JSONTreeNode]?) throws -> Bool
```

Reply:
Because there are no codes that have contents nil in TC, I modified the code to show the error to you. 
You can see the errors if you change "any" to "some" in edit() that similar editByPath(). (and check editInternal() too)

Reply:
Thank you for your clarification. I misunderstood the use case of the function.